id.go

```diff
@@ -101,9 +100,7 @@ func init() {
 func readMachineID() []byte {
 	id := make([]byte, 3)
 	if hostname, err := os.Hostname(); err == nil {
-		hw := md5.New()
-		hw.Write([]byte(hostname))
-		copy(id, hw.Sum(nil))
+		copy(id, []byte(hostname))
```

Comment:
What is the reasoning here?

Reply:
I was trying to see what was the point of the md5 step? Could we skip it? Or is it to make sure you can fill the machine id bytes with non zero value?

Reply:
It is easy to imagine a cluster in which hostnames start with the same 3 bytes (eg: web001, web002, ...).

Reply:
ok! I'll switch back. Do you mind me adding the error handling in the "Write" though?

Reply:
How could that error condition happen?